### ISSUE TITLE
[wiki] Gitbook and Harmony Puzzle dead links in the wiki

### ISSUE THREAD

**Issue by Mouradif** [harmony-one/harmony#3609]
**Describe the bug**

I don't know if it's intended but from the perspective of an enthusiast reading about the project, it sure looks like a bug. The documentation is full of broken links (or non-public links) to gitbook, and the link on the side (Edit on GitHub) point to the unavailable `web-wiki` repo (https://github.com/harmony-one/web-wiki/). Here's a list I've collected of broken links found:

- [ ] https://wiki.harmony.one/papers-and-social-links#wikis the 2 links "Node runners" and "Developers"
- [ ] https://wiki.harmony.one/community/run-a-harmony-node/run-a-node All 13 links
- [ ] https://wiki.harmony.one/community/run-a-harmony-node/pangaea#get-involved "Gitbook"
- [ ] https://wiki.harmony.one/tech-zone/tech-deep-sharding#sdk both links "Harmony SDK" and "here"
- [ ] https://wiki.harmony.one/tech-zone/dapps/harmony-puzzle The link to the playable game is dead (DNS doesn't resolve)
- [ ] https://wiki.harmony.one/tech-zone/sdk All 27 links

So basicly everything I need to know to start building on Harmony, use the SDK, run a node, is behind a wall

**Comment by yelllowsin** [comment]
Hi @Mouradif we migrated the docs to https://docs.harmony.one. You will find there everything you need from building on Harmony, use the SDK, run a node etc.

Aligning already with the team so that we archive this old space and migrate any remaining doc the the link above.

**Comment by yelllowsin** [comment]
I reviewed the Wiki content and decided to archive everything as I did not find anything worth migrating. The only interesting section was the Technology one, but most important things related to Harmony functionality are already on docs [here](https://docs.harmony.one/home/general/technology). I put a notice with links redirecting to official docs and discord the same way I did in the past with other similar spaces.
